### PR TITLE
standardized naming conventions across dir

### PR DESCRIPTION
**Pull request by emiliocramer**
Moving onto dummy chain DMLOG prefix -> FIRE prefix

### CODE REVIEW

File: cmd/fireacme/cli/node.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func nodeFactoryFunc(flagPrefix, kind string) func(*launcher.Runtime) (launcher.
 		nodeDataDir := replaceNodeRole(kind, mustReplaceDataDir(sfDataDir, viper.GetString(flagPrefix+"data-dir")))
 
 		readinessMaxLatency := viper.GetDuration(flagPrefix + "readiness-max-latency")
-		debugDeepMind := viper.GetBool(flagPrefix + "debug-deep-mind")
+		debugFirehose := viper.GetBool(flagPrefix + "debug-deep-mind")
```
Comment (maoueh) [inline cmd/fireacme/cli/node.go]: This feels bad now, and the flag needs to change. Also you will need to validate this works correctly, I can help with that.

File: devel/standard/standard.yaml [hunk 2]
```diff
@@ -2,13 +2,13 @@ start:
...
-  - extractor-node
+  - reader-node
   - relayer
   flags:
     # Specifies the path to the binary
-    extractor-node-path: "dummy-blockchain"
+    reader-node-path: "dummy-blockchain"
     # Flags that will be added to the dummy chain process command
-    extractor-node-arguments: +--dm-enabled --block-rate=6
+    reader-node-arguments: +--dm-enabled --block-rate=6
```
Comment (maoueh) [inline devel/standard/standard.yaml]: `dm-enabled` flag in `dummy-blockchain` will need to change to `firehose-enabled` also (use the flag as defined in the conventions doc).

File: codec/consolereader_test.go [hunk 3]
```diff
@@ -31,21 +31,21 @@ import (
 
 func TestParseFromFile(t *testing.T) {
 	tests := []struct {
-		deepMindFile     string
+		firehoseFile     string
 		expectedPanicErr error
 	}{
 		{"testdata/deep-mind.dmlog", nil},
```
Comment (maoueh) [inline codec/consolereader_test.go]: Let's rename the actual file also to be named `full.firelog`

File: codec/consolereader_test.go [hunk 4]
```diff
@@ -31,21 +31,21 @@ import (
 
 func TestParseFromFile(t *testing.T) {
 	tests := []struct {
-		deepMindFile     string
+		firehoseFile     string
```
Comment (maoueh) [inline codec/consolereader_test.go]:
```suggestion
		firehoseLogsFile     string
```

File: cmd/fireacme/cli/main.go [hunk 5]
```diff
@@ -81,7 +81,7 @@ func Main() {
 	derr.Check("acme-blockchain", RootCmd.Execute())
 }
 
-var startCmdExample = `fireacme start extractor`
+var startCmdExample = `fireacme start reader`
```
Comment (maoueh) [inline cmd/fireacme/cli/main.go]: Pretty sure this should be
```suggestion
var startCmdExample = `fireacme start reader-node`
```

Test it to see.

File: cmd/fireacme/cli/node.go [hunk 6]
```diff
@@ -24,15 +24,15 @@ import (
...
+var readerAcmeChainLogger, _ = logging.PackageLogger("reader.acme", "github.com/streamingfast/firehose-acme/reader/acme", DefaultLevelInfo)
 
 func registerCommonNodeFlags(cmd *cobra.Command, flagPrefix string, managerAPIAddr string) {
 	cmd.Flags().String(flagPrefix+"path", ChainExecutableName, FlagDescription(`
 		Process that will be invoked to sync the chain, can be a full path or just the binary's name, in which case the binary is
 		searched for paths listed by the PATH environment variable (following operating system rules around PATH handling).
 	`))
-	cmd.Flags().String(flagPrefix+"data-dir", "{data-dir}/{node-role}/data", "Directory for node data ({node-role} is either extractor, peering or dev-miner)")
+	cmd.Flags().String(flagPrefix+"data-dir", "{data-dir}/{node-role}/data", "Directory for node data ({node-role} is either reader, peering or dev-miner)")
 	cmd.Flags().Bool(flagPrefix+"debug-deep-mind", false, "[DEV] Prints deep mind instrumentation logs to standard output, should be use for debugging purposes only")
```
Comment (maoueh) [inline cmd/fireacme/cli/node.go]: Need to change comment and flag. For the flag, let's use `...+"debug-firehose-logs` for now even though fully satisfy by the name.

I think there might be other references to `deep mind` maybe (search for `deep` and `mind` separately to find outliers).

File: cmd/fireacme/cli/node.go (hunk 1 above)
Comment (maoueh) [reply cmd/fireacme/cli/node.go]: `debug-firehose-logs`

**Review by maoueh — CHANGES_REQUESTED** [review]

File: devel/standard/standard.yaml (hunk 2 above)
Comment (emiliocramer) [reply devel/standard/standard.yaml]: yes I saw that in the dummy, didnt know for sure. Thanks

**Pull request merged**